$NetBSD: patch-ac,v 1.1.2.2 2002/06/23 18:54:50 jlam Exp $

Need to include sys/types.h so that pid_t is a defined type for
sys/select.h.

--- src/internal.h.orig	Sat Sep 16 20:24:24 2000
+++ src/internal.h
@@ -30,6 +30,8 @@
 #ifndef ADNS_INTERNAL_H_INCLUDED
 #define ADNS_INTERNAL_H_INCLUDED
 
+#include <sys/types.h>
+
 #include "config.h"
 typedef unsigned char byte;
 
